src/plexil-adapter/OwAdapter.cpp

```diff
@@ -52,7 +44,22 @@ static vector<Value> const EmptyArgs;
 
 static int CommandId = 0;
 
-static std::map<int, pair<Command*, AdapterExecInterface*>> CommandRegistry;
+std::mutex g_shared_mutex;
```

Comment:
Can this mutex variable be static?

Reply:
Ops! Just saw this after I merged. It doesn't make a lot of a difference for how the variable is going to be used.